pkg/commands/render.go

```diff
@@ -133,7 +133,7 @@ func makeRTEObjects(commonOpts *CommonOptions) ([]client.Object, string, error)
 	if err != nil {
 		return nil, namespace, err
 	}
-	mf = mf.Update(rtemanifests.UpdateOptions{
+	mf = mf.Update(&rtemanifests.UpdateOptions{
```

Comment:
why the pointer?

Reply:
same as with new, we will create two objects if we will not pass the pointer:
1. rtemanifests.UpdateOptions{...}
2. copy of rtemanifests.UpdateOptions{...} under the method

Reply:
let's postpone this change in another PR. Let's keep copy by value for the time being.